src/main/java/net/dv8tion/jda/api/utils/messages/MessageCreateBuilder.java

```diff
@@ -268,7 +264,7 @@ public MessageCreateBuilder clear()
         super.clear();
         this.files.clear();
         this.tts = false;
-        this.silent = false;
+        messageFlags &= ~Message.MessageFlag.NOTIFICATIONS_SUPPRESSED.getValue();
```

Comment:
Could you do `messageFlags = 0` in `AbstractMessageBuilder#clear` instead? The flags should also be handled in `MessageCreateRequest#applyData` and `MessageCreateRequest#applyMessage`.

Reply:
Done.